documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -0,0 +1,235 @@
+
+- The logic for including a 30k deliverable in reporting should be clear and easy to understand
+- Users should be able to add or remove a given 30k deliverable from reports without having to change the underlying logic
+- If necessary, users should be able to change the reporting logic without rewriting significant sections of the source code
+
+## Options considered
+
+### Assigning issues to deliverables
+
+- **Milestones:** Add the issue to a milestone that represents a 30k deliverable
+- **Labels:** Tag the issue with a label that represents a 30k deliverable
+- **Deliverable column:** Add a custom "deliverable" column to the GitHub projects with values representing each 30k deliverable
+- **Reserved phrase:** Use a reserved phrase to tag a 30k deliverable in the body of the issue
+
+### Limiting the 30k deliverables included in reporting
+
+- **Label:** Use a label to indicate when a deliverable should be included in reporting
+- **Status:** Use the status of the deliverable in the provisional roadmap project
+- **GitHub action:** Explicitly list the deliverables to include in the GitHub action for reporting
+
+## Decision outcomes <!-- REQUIRED -->
+
+### Assigning issues to deliverables
+
+We've decided to use a "deliverable" column to assign issues to a parent 30k deliverable. This involves:
+
+- creating a single select column in both the product roadmap and the sprint board
+- populating that column with options that represent each 30k deliverables in our roadmap
+- using that column to indicate the 30k deliverable that an issue is assigned to
+
+- **Positive outcomes**
+  - We can filter, group, and sort issues by deliverable in GitHub projects and GitHub's automated insight reporting.
+  - The engineering team can continue to use GitHub milestones to organize issues into units of work that are smaller than a 30k deliverable.
+- **Negative outcomes**
+  - We won't be able to filter for issues assigned to a given 30k deliverable within the GitHub repository.
+  - We'll have to make sure the list of options in the deliverable column is consistent across GitHub projects in order to join issues from different projects correctly in our custom reporting.
+  - We'll have to update the current logic in our custom reporting so that issues are joined to their parent deliverable using the value of this column.
+
+> [!NOTE]
+> If the team finds a consistent need to filter the list of issues by deliverable within the repository, we may revisit this decision and choose to use ***both*** a label and a deliverable column to assign an issue to a 30k deliverable. We've decided **not** to use both options for the time being to avoid having multiple (potentially conflicting) ways of assigning an issue to a given deliverable.
+
+### Limiting the 30k deliverables included in reporting
+
+We've decided to use the status of the deliverable in the product roadmap board to determine when deliverables are included or excluded from reporting.
```

Comment:
Makes sense to me! 

We can always make "custom" reports on certain deliverables too -- e.g., here's the graph of status, and oh also, Deliverable X is now 100% done and Deliverable Y is 10% done too.